.github/workflows/release.yml

```diff
@@ -5,113 +5,146 @@ on:
           build-args: |
             COMMIT=${{ github.sha }}
             BUILD_DATE=${{ env.BUILD_DATE }}
-            RUN_NUMBER=${{ github.run_number }}
+            RUN_NUMBER=${{ env.RUN_NUMBER }}
             TAG=${{ env.RELEASE_VERSION }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
 
-      ### Create GitHub release
-      - name: Build Changelog
+  release:
+    name: "Release"
+    runs-on: ubuntu-latest
+
+    needs:
+      - prepare
+      - build
+
+    env:
+      RELEASE_VERSION: ${{ needs.prepare.outputs.release_version }}
+
+    steps:
+      - name: Build changelog
         id: github_release
-        uses: mikepenz/release-changelog-builder-action@v1
+        uses: mikepenz/release-changelog-builder-action@v3
         with:
-          configuration: ".github/workflows/changelog_config.json"
+          configuration: ".github/misc/changelog_config.json"
           commitMode: true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Parse metadata from docker publish
-        id: metadata
-        uses: actions/github-script@v4
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          result-encoding: string
-          script: |
-            try {
-                let metadataRaw = `${{ steps.docker_publish.outputs.metadata }}`;
-                let metadata = JSON.parse(metadataRaw);
-                let imageTags = metadata['image.name'].split(",");
-                let tagString = imageTags.reduce(
-                  (acc, tag) => { return `${acc}\n- ${tag}` },
-                  ""
-                )
-                console.log(tagString);
-                return tagString;
-            } catch (e) {
-              console.log("Err: " + e);
-              throw e;
-            }
-
       - name: Release
-        id: release
         uses: softprops/action-gh-release@v1
         with:
           name: Fluence Node ${{ env.RELEASE_VERSION }}
           tag_name: ${{ env.RELEASE_VERSION }}
           body: |
-            ${{steps.github_release.outputs.changelog}}
+            ${{ steps.github_release.outputs.changelog }}
 
             ## Docker tags
-            ${{steps.metadata.outputs.result}}
+
+            ### ipfs
+
+            fluencelabs/fluence:latest
+            fluencelabs/fluence:${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:ipfs
+            fluencelabs/fluence:ipfs_${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:ipfs_${{ env.RELEASE_VERSION }}_${{ env.RUN_NUMBER }}
+
+            ## minimal
+
+            fluencelabs/fluence:minimal
+            fluencelabs/fluence:minimal_${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:minimal_${{ env.RELEASE_VERSION }}_${{ env.RUN_NUMBER }}
+
+            ## rich
+
+            fluencelabs/fluence:rich
+            fluencelabs/fluence:rich_${{ env.RELEASE_VERSION }}
+            fluencelabs/fluence:rich_${{ env.RELEASE_VERSION }}_${{ env.RUN_NUMBER }}
```

Comment:
let's add some description here maybe? Like
```
### ipfs

Has Fluence Rust Peer, and IPFS daemon as a sidecar
...

### minimal

Contains only Fluence Rust Peer (no aqua-ipfs, no ceramic adapter)

###

Contains Fluence Rust Peer, IPFS daemon as a sidecar, ceramic cli, bitcoin cli and geth cli

...
```

Also I'd add a table to README like
| Container flavor  | Fluence Rust Peer | IPFS daemon | aqua-ipfs | registry | binaries |
| ---------------   | ---------------   | ----------- |----------- |----------- |----------- |
| minimal           |        ✅         | ❌           | ❌        |  ✅       | curl     |
| ipfs              |        ✅         | ✅           | ✅        |  ✅       | curl, ipfs |
| rich              |        ✅         | ✅           | ✅        |  ✅       | curl, ipfs, ceramic, bitcoin cli, geth cli |

Reply:
Actually aqua-ipfs present even in the minimal container. Need to redo builtins installation script to be able to choose which services to install. Better do to it separately in another task.